static/openapi.json

```diff
@@ -17,7 +17,18 @@
               "type": "string"
             },
             "name": "bibkeys",
-            "in": "query"
+            "in": "query",
+            "examples": {
```

Comment:
In #5910, we will need to add this into `openapi_dict` before we write the json or yaml output...
```python
openapi_dict["paths"]["/api/books"]["get"]['parameters"]["examples"] = "xyz"
```